doc/man/8/ceph.rst

```diff
@@ -369,11 +400,25 @@ Usage::
 
 Subcommand ``new`` to make a new file system using named pools <metadata> and <data>
 
+Subcommand ``lsflags`` prints all the flags set on the given FS.
```

Comment:
Subcommand ``lsflags`` displays ...

Reply:
... and ditto for other commands too.

Reply:
Changed.

Why not print? It felt more unambiguous to me. :)